opengrok-indexer/src/test/java/org/opengrok/indexer/util/FileUtilities.java

```diff
@@ -53,11 +53,10 @@ public static void extractArchive(File sourceBundle, File root) throws IOExcepti
             if (ze.isDirectory()) {
                 file.mkdirs();
             } else {
-                InputStream in = zipfile.getInputStream(ze);
-                assertNotNull(in);
-                FileOutputStream out = new FileOutputStream(file);
-                assertNotNull(out);
-                copyFile(in, out);
+                try (InputStream in = zipfile.getInputStream(ze); FileOutputStream out = new FileOutputStream(file)) {
+                    assertNotNull(in);
```

Comment:
Can’t we do better than assert? IOexception?

Reply:
We could. But I told myself that it's just for tests so why bother :D

Reply:
But honestly I did not like the assert as well :) I can change it when I get home.